fe2-android/app/src/main/java/com/github/dedis/popstellar/ui/detail/event/election/fragments/CastVoteFragment.java

```diff
@@ -45,10 +45,11 @@ public class CastVoteFragment extends Fragment {
         for (int i = 0; i < electionQuestions.size(); i++) {
           ElectionQuestion electionQuestion = electionQuestions.get(i);
           List<Integer> votes = mLaoDetailViewModel.getCurrentElectionVotes().getValue().get(i);
+          // Only one vote should be selected.
           ElectionVote electionVote =
               new ElectionVote(
                   electionQuestion.getId(),
-                  votes,
+                  votes.get(0),
```

Comment:
Why do you get only the vote of the 1 st question ?

Reply:
because now the vote is unique, even if the user selects multiple questions, it will discard them and take only one vote, the first.

Reply:
Just to clarifiy to be sure there is no misunderstanding : 

The user can't select multiple questions, he can vote on (for now) 1 ballot option per question. The organizer when he sets up the election sets the number of questions (>=1) and the number (>=2) and content of ballots options. Of course for the plurality vote the user cannot vote for more than one option per question. 
The multiple question thing is like having the possibility to vote for national and regional elections in the same event. Each question (in this case could regional or national) has its set of ballots options, and the user can vote for one option of each

Reply:
I specifically it for the new cast vote protocol here https://github.com/dedis/popstellar/pull/918. The ElectionVote now contains only one integer instead of of list of integers. Do you think my implementation has a problem ?

Reply:
I think so. It is more subtle than I realised at first. It is probably that the current implementation requires more adaptations to fit the new protocol.  I think that we should discuss in more details. I don't have time until tomorrow afternoon unfortunately.

Reply:
i will be glad to discuss it yeah !